WordPress/Classes/Services/PeopleService.swift

```diff
@@ -263,9 +263,199 @@ struct PeopleService {
     }
 }
 
+// MARK: - Invite Links Related
+
+extension PeopleService {
+
+    /// Convenience method for retrieving invite links from core data.
+    ///
+    /// - Parameters:
+    ///   - siteID: The ID of the site.  The site should be a P2.
```

Comment:
> The site should be a P2.

Should it? I know we'll add the restriction in the UI. But fetching, generating, and disabling invites all work for a non-P2 site. And there aren't checks in any of these methods to force a P2. I just find these comments a bit misleading/confusing.

Reply:
Good points. I'm game to drop that comment.